### PR TITLE
Added INWX provider

### PR DESCRIPTION
**Pull request by greaka**

### CODE REVIEW

**Comment by greaka** [comment]
What is missing to merge this PR?

File: inwx/inwx.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	caddytls.RegisterDNSProvider("inwx", NewDNSProvider)
+}
+
+// NewDNSProvider returns a new INWX DNS challenge provider.
+// The credentials are interpreted as follows:
+//
+// len(0): use credentials from environment
+// len(3): credentials[0] = Base URL
+//         credentials[1] = User ID
+//         credentials[2] = Key
```
Comment (mholt) [inline inwx/inwx.go]: Where is credentials[2] used?

**Comment by greaka** [comment]
forgot to update the comment on the usage of the credentials when copying the initial script from auroradns

fixed it

**Comment by greaka** [comment]
@mholt is anything still missing?

**Pull request merged**